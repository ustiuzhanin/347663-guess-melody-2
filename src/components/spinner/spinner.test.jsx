import React from "react";
import renderer from "react-test-renderer";
import Spinner from "./spinner.jsx";

test(`Spinner's Snapshot`, () => {
  const tree = renderer.create(<Spinner />).toJSON();

  expect(tree).toMatchSnapshot();
});
