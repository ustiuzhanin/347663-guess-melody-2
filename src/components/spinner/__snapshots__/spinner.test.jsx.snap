// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner's Snapshot 1`] = `
<img
  alt="preloader"
  height={100}
  src="img/loading_spinner.gif"
  width={100}
/>
`;
